lib/matplotlib/font_manager.py

```diff
@@ -986,7 +986,13 @@ def __init__(self, size=None, weight='normal'):
         for fontext in ["afm", "ttf"]:
             for path in [*findSystemFonts(paths, fontext=fontext),
                          *findSystemFonts(fontext=fontext)]:
-                self.addfont(path)
+                try:
+                    self.addfont(path)
+                except OSError as exc:
+                    _log.info("Failed to open font file %s: %s", path, exc)
+                except Exception as exc:
+                    _log.info("Failed to extract properties from %s: %s",
```

Comment:
```suggestion
                    _log.info("Failed to extract font properties from %s: %s",
```
To give context to the message.

Reply:
done